pkg/qe-tests/cypress/integration/tests/tier1/vmware/tier1_tests_mapping.ts

```diff
@@ -0,0 +1,55 @@
+import { ProviderVmware } from '../../../models/providerVmware';
+import {
+  clickByText,
+  createNamespace,
+  getStarted,
+  login,
+  provisionNetwork,
+} from '../../../../utils/utils';
+import { vmwareTier1TestCephCold } from './tier1_config_vmware';
+import {
+  vmwareNetworkMapping_2x_network,
+  vmwareNetworkMappingSingle,
+  vmwareProviderUser,
+} from './tier1_config_vmware';
+import { button, SEC } from '../../../types/constants';
+import { MappingNetwork } from '../../../models/mappingNetwork';
+// import { MappingStorage } from '../../../models/mappingStorage';
+
+describe('Tier1 tests for testing provider', () => {
+  const provider = new ProviderVmware();
+  const networkMapping = new MappingNetwork();
+  // const storageMapping = new MappingStorage();
+  before('Login to MTV', () => {
+    // Clearing all cookies in local storage if any
+    cy.clearLocalStorageSnapshot();
+    login(vmwareTier1TestCephCold.loginData);
+    getStarted();
+    provider.create(vmwareProviderUser);
+    // Saving local storage state after login
+    cy.saveLocalStorage();
+    createNamespace(vmwareTier1TestCephCold.planData.namespace);
+    provisionNetwork(vmwareTier1TestCephCold.planData.namespace);
+  });
+
+  beforeEach('Login to MTV', () => {
+    // Restoring local storage and opening base MTV URL
+    cy.restoreLocalStorage();
+    cy.visit(vmwareTier1TestCephCold.loginData.url, { timeout: 120 * SEC });
+    clickByText(button, 'Get started');
+  });
+
+  it('Creating and editing network mapping', () => {
```

Comment:
I think the test description would make sense saying "Creates and edits network mapping".
It a small thing but help when reviewing tests.

Reply:
I guess this is exactly the same meaning, this description is shown when test is executed, and it looks better when I see my variant in the test progress